frontend/src/components/SideBar.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import HomeIcon from './icons/HomeIcon'
+import ArtistIcon from './icons/ArtistIcon'
+import TracksIcon from './icons/TracksIcon'
+import SetlistIcon from './icons/SetlistIcon'
+
+export default class Header extends Component {
```

Comment:
Syntax of this should be consistent with other components: try keeping it like this across the board
`const Header: FC = () => {}
export default Header
`